litmus-portal/frontend/src/components/ProfileDropdown/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useRef, useState, useCallback } from 'react';
+import { IconButton, Grid } from '@material-ui/core';
+import Avatar from '@material-ui/core/Avatar';
+import ExpandMoreTwoToneIcon from '@material-ui/icons/ExpandMoreTwoTone';
+import Typography from '@material-ui/core/Typography';
+import ProfileInfoDropdownItems from '../ProfileDropdownItems';
+import useStyles from './styles';
+
+const ProfileDropdown = ({ callbackFromParent }: any) => {
+  localStorage.setItem('User', 'Richard Hill');
```

Comment:
no need to set and get from local storage in the same component...we will add this if required...use state variable or constant variables if you want to use dummy data